src/components/app-rating-mobile/AppRatingMobile.tsx

```diff
@@ -9,18 +9,41 @@ import { styles } from '~/components/app-rating-mobile/AppRatingMobile.styles'
 
 interface AppRatingMobileProps extends RatingProps {
   reviewsCount: number
+  linkHash?: ReactElement
+  style?: {
+    variantOption?: string
+    fontSize?: string
+    starMobile?: object
+    reviews?: object
+    rating?: object
+  }
 }
 
-const AppRatingMobile: FC<AppRatingMobileProps> = ({ value, reviewsCount }) => {
+const AppRatingMobile: FC<AppRatingMobileProps> = ({
+  value,
+  reviewsCount,
+  linkHash,
+  style
+}) => {
   const { t } = useTranslation()
 
   return (
     <Box sx={styles.root}>
       <Box data-testid='number-box' sx={styles.number}>
-        <StarSharp data-testid='star-icon' sx={styles.starMobile} />
-        <Typography variant={'h6'}>{value}</Typography>
+        <StarSharp
+          data-testid='star-icon'
+          sx={style?.starMobile ? style?.starMobile : styles.starMobile}
```

Comment:
use spiceSx func, if u want to pass some styles, you don't need to pass all fields , but only those you want to change

Reply:
Used.